db/migrate/20151023131950_link_arf_report_directly_to_host.rb

```diff
@@ -0,0 +1,17 @@
+class LinkArfReportDirectlyToHost < ActiveRecord::Migration
+  def up
+    ForemanOpenscap::ArfReport.all.each do |report|
```

Comment:
I've tried this on my db - a few comments:

* please never try to loop on large tables such as reports, if you do, use find in batches instead.
* the logic here is odd, report already has a host_id, and then you are using that id to find the asset and save it again on the report host_id?
* third, please use update_all instead this will completely avoid the each loop.